docker-compose.yml

```diff
@@ -1,5 +1,13 @@
 version: "3"
 services:
   mongo:
-    network_mode: host
     image: mongo
+
+  bot:
+    build: .
+    env_file:
+      .env
+    links:
+      - "mongo:mongo"
+    ports:
+      - 32000:31337
```

Comment:
So this docker compose file was meant to be used in development. So this would contain all the services that the bot would need to run but we would still run the bot outside of docker. We can use `yarn dev` to run the service with hot reload which is also waay quicker than using docker (as the builds are slower) meaning we get a faster feed back cycle.